website/docs/d/kusto_database.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Data Explorer"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_kusto_database"
+description: |-
+  Manages Kusto / Data Explorer Database
+---
+
+# Data Source: azurerm_kusto_database
+
+Use this data source to access information about an existing Kusto (also known as Azure Data Explorer)  a Kusto (also known as Azure Data Explorer) Database
+
+## Example Usage
+
+```hcl
+resource "azurerm_kusto_database" "database" {
+  name                = "my-kusto-database"
+  resource_group_name = "test_resource_group"
+  cluster_name        = "test_cluster"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - The name of the Kusto Database to create.
+
+* `resource_group_name` - Specifies the Resource Group where the Kusto Database should exist.
+
+* `cluster_name` - Specifies the name of the Kusto Cluster this database will be added to.
+
```

Comment:
```
## Attributes Reference

The following attributes are exported:
```
List the computed properties, like https://github.com/heiazuo/terraform-provider-azurerm/blob/main/website/docs/d/kusto_cluster.html.markdown#attributes-reference